src/main/java/frc/robot/commands/SwerveCommands/AlignToSpeaker.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.SwerveCommands;
+
+import static frc.robot.Constants.IntakeArm.minimumError;
+import static frc.robot.Constants.Swerve.minimumErrorTurn;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.Utils.vision.Vision;
+import frc.robot.commands.SwerveCommands.*;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class AlignToSpeaker extends Command {
+  /** Creates a new TurnToDegree. */
+  SwerveSubsystem swerve = SwerveSubsystem.getInstance();
+  private PIDController pid = new PIDController(0, 0, 0);
```

Comment:
put it in a constant called aliging PID 

put the entire pid in a constant not a KP KI KD